.gitignore

```diff
@@ -98,7 +98,7 @@ celerybeat-schedule
 .env
 
 # virtualenv
```

Comment:
Why do we need * here ?

Reply:
This is just to ignore virtual environment folders created such as `.venv3.11` or `.venv3.10` instead of just `.venv`.

Reply:
This is just for testing purpose. We can name the virtualenv to anything we'd like.